client/blocks/upsell-nudge/index.jsx

```diff
@@ -104,6 +108,12 @@ export const UpsellNudge = ( {
 		{ 'is-wpcom-plan': plan && planMatches( plan, { group: GROUP_WPCOM } ) }
 	);
 
+	if ( dismissPreferenceName && forceHref && href ) {
+		debug(
+			"It's not possible to force the whole banner to be a link when it needs to link and be dismissable"
+		);
+	}
+
```

Comment:
Wouldn't make more sense to place this in `SidebarBannerTemplate` which is the place where we ignore the `forceHref` prop if the nudge is dimissible?

Reply:
Maybe this could be improved but I'm not sure this is it. `SidebarBannerTemplate` doesn't receive forceHref as a prop, it just gives a sane forceHref prop to `UpsellNudge`, which logs the above message in case someone else ever gives it an incompatible set of props.

It's actually the `Banner` components getContent method which does the ignoring, if called to provide the content for a `DismissibleCard`, so perhaps that ought to have a debug message too.